cmd/gapit/screenshot.go

```diff
@@ -60,9 +61,17 @@ func (verb *screenshotVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 		return nil
 	}
 
-	filepath, err := filepath.Abs(flags.Arg(0))
+	captureFilepath := flags.Arg(0)
+
+	// Try parsing captureFilepath as a hex string.
+	captureId, err := id.Parse(captureFilepath)
```

Comment:
`captureID` - govet will moan about this.
Also - it might be nice to wrap all the verbs that take a capture path argument with a helper function that does this logic.